src/main/java/io/confluent/examples/streams/WikipediaFeedAvroExample.java

```diff
@@ -129,6 +130,11 @@ static KafkaStreams buildWikipediaFeed(final String bootstrapServers,
     // in order to keep this example interactive.
     streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 10 * 1000);
 
+    // if Confluent monitoring interceptors are found on the classpath
+    // then the producer and consumer interceptors are added to the
+    // streams application if not then no action is taken
```

Comment:
1. Missing punctuation before "if not"
2. "if not then no action is taken" -- is inaccurate.  A message is printed.

Reply:
ack